networkx/algorithms/operators/binary.py

```diff
@@ -137,15 +137,13 @@ def disjoint_union(G, H):
 
 
 def intersection(G, H):
-    """Returns a new graph that contains only the edges that exist in
+    """Returns a new graph that contains only the nodes and the edges that exist in
     both G and H.
 
-    The node sets of H and G must be the same.
-
     Parameters
     ----------
     G,H : graph
-       A NetworkX graph.  G and H must have the same node sets.
+       A NetworkX graph. G and H can have different node sets but must be both graphs or both multigraphs.
```

Comment:
Could one be a MultiGraph and the other be a Graph?
What happens in that case?

Reply:
It raises a NetworkX error, I didn't find any straightforward definition for the intersection of graphs of different types.